syntax="proto2";

import "goby/zeromq/protobuf/liaison_config.proto";

package protobuf;

extend goby.apps.zeromq.protobuf.LiaisonConfig
{
    optional LiaisonNetsimConfig netsim_config = 1300;
}


message LiaisonNetsimConfig
{
    //    optional string image_dir = 1;
}
